go-controller/pkg/ovn/master.go

```diff
@@ -606,12 +606,12 @@ func (oc *DefaultNetworkController) syncNodes(kNodes []interface{}) error {
 
 	staleNodes := sets.NewString()
 	for _, nodeSwitch := range nodeSwitches {
-		if !foundNodes.Has(nodeSwitch.Name) {
+		if nodeSwitch.Name != types.TransitSwitch && !foundNodes.Has(nodeSwitch.Name) {
```

Comment:
how about moving this `nodeSwitch.Name != types.TransitSwitch` to be part of `defaultNetworkPredicate` ?

Reply:
also, please add a test case for transit into the unit test:
`func TestController_syncNodes`:  https://github.com/ovn-org/ovn-kubernetes/blob/017427db418a8491edfd2088de04943881af14b7/go-controller/pkg/ovn/master_test.go#L1655
?

Reply:
I don't think it has any effect. If anything I would have to change the predicate name variable, and the logic in the predicate is not obvious without the name of the variable. I'm just going to leave it as is.